torch/_inductor/runtime/hints.py

```diff
@@ -112,7 +112,7 @@ def create(cls, device):
 
         device_type = device.type if torch.version.hip is None else "hip"
         device_interface = get_interface_for_device(device)
-        if device_type == "cuda":
+        if any(accepted_type in device_type for accepeted_type in ["cuda", "hip"]):
```

Comment:
nit: 
```
if device_type in ["cuda", "hip"]:
```

Reply:
Thanks, I thought that I tried this way initially but was hitting some dynamo error. I must have gotten it the wrong way around seems to work fine.

There's a bit of cleanup needed in this PR let me re-ping when I'm ready to move from draft